functests/performance/updating_profile.go

```diff
@@ -133,6 +136,19 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("Reverts back all profile configuration", func() {
+			// BUG: CNF-385. Workaround - we have to remove hugepages first
```

Comment:
is this still an issue with https://github.com/kubernetes/kubernetes/pull/82820 being merged in k8s 1.18 and OCP being rebased? (did not look into details, just asking)

Reply:
@cynepco3hahue do you know? :)

Reply:
it requires to enable the specific feature gate until this PR(https://github.com/openshift/api/pull/604) is merged the feature still not usable